.github/workflows/checks.yml

```diff
@@ -21,8 +21,10 @@ jobs:
           go-version: '1.20.x'
       - uses: actions/checkout@v3
 
+      # Temporarily locking prettier at 2.8.8. See:
+      # https://github.com/AthanorLabs/atomic-swap/issues/501
       - name: Install npm formatting/linting tools
-        run: npm install --global --save-dev prettier prettier-plugin-solidity solhint
+        run: npm install --global --save-dev prettier@2.8.8 prettier-plugin-solidity solhint
```

Comment:
I didn't update the documentation. The Solidity plugin [has an open PR](https://github.com/prettier-solidity/prettier-plugin-solidity/pull/848) that presumably will fix the issue. They just need to merge it and make a release. Given that, I think we can remove the version peg before anyone will need to modify the Solidity code.

Reply:
Thanks. I implemented it.